pkg/webhook/pod/validating/cluster_colocation_profile.go

```diff
@@ -68,6 +69,24 @@ func (h *PodValidatingHandler) clusterColocationProfileValidatingPod(ctx context
 	return allowed, reason, nil
 }
 
+var forbidAnnotations = []string{
+	extension.AnnotationReservationAllocated,
+}
+
+func forbidSpecialAnnotations(pod *corev1.Pod) field.ErrorList {
+	if pod.Annotations == nil {
+		return nil
+	}
+	els := field.ErrorList{}
```

Comment:
`els` is not a good name, looks like `else`, please rename to `errors` or `errorList`.